backend/corpora/lambdas/api/v1/collection.py

```diff
@@ -32,26 +32,27 @@ def get_collections_list(from_date: int = None, to_date: int = None, user: Optio
     return make_response(jsonify(result), 200)
 
 
-@db_session()
 def get_collection_details(collection_uuid: str, visibility: str, user: str):
-    collection = Collection.get_collection(collection_uuid, visibility)
-    if collection:
-        if user == collection.owner:
-            access_type = "WRITE"
-        elif visibility != "PUBLIC":
-            raise ForbiddenHTTPException()
-        else:
-            access_type = "READ"
-        result = collection.reshape_for_api()
-        result["access_type"] = access_type
-        return make_response(jsonify(result), 200)
-    else:
+    session = g.db
```

Comment:
So `session` means one thing in flask and another thing in sqlalchemy. It could a little confusing pull the sqlalchemy meaning out of a flask variable.

Reply:
Since we are not using a flask session, I thought it was ok to use that variable name. I can rename it to db_session for clarity.